packages/react-components/src/components/chart/baseChart.tsx

```diff
@@ -65,7 +68,7 @@ const BaseChart = ({ viewport, queries, size = { width: 500, height: 500 }, ...o
     useResizeableEChart(chartRef, size);
 
   // apply group to echarts
-  useGroupableEChart(chartRef, options.groupId);
+  useGroupableEChart(chartRef, group);
```

Comment:
`group ?? options.groupId` in case the viewport group is nonexistent?

Reply:
groupId will not be part of options i.e props anymore. if viewport does not exist, we are simply not in sync mode. and the undefined value of the group is handled in the hook